Frends.LinkedIn.SearchAdCreatives/Frends.LinkedIn.SearchAdCreatives/LinkedIn.cs

```diff
@@ -0,0 +1,186 @@
+﻿namespace Frends.LinkedIn.SearchAdCreatives
+{
+    using System;
+    using System.Collections.Generic;
+    using System.ComponentModel;
+    using System.Linq;
+    using System.Net;
+    using System.Net.Http;
+    using System.Net.Http.Headers;
+    using System.Runtime.Caching;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using System.Web;

```

Comment:
This namespace is not used, so it can be removed.

Reply:
The namespace is used with HttpUtility class